wallet/src/de/schildbach/wallet/ui/MainActivity.kt

```diff
@@ -150,11 +167,6 @@ class MainActivity : AbstractBindServiceActivity(), ActivityCompat.OnRequestPerm
         viewModel.showCreateUsernameEvent.observe(this, {
             startActivity(Intent(this, CreateUsernameActivity::class.java))
         })
-        if (walletApplication.configuration.developerMode) {
-            viewModel.inviteData.observe(this, {
-                inviteHandler.handle(it)
-            })
-        }
```

Comment:
Handling invites is no longer considered in developer mode?  That is probably a good idea since this PR is the last one to finally finish the UI related work for it.

The backend needs one change, but that will included with v0.20 changes in `android-dashpay`